packages/runtime/container-runtime-definitions/src/containerRuntime.ts

```diff
@@ -82,7 +82,7 @@ export interface IContainerRuntime extends
      * @param id - Id supplied during creating the data store.
      * @param wait - True if you want to wait for it.
      */
-    getRootDataStore(id: string, wait?: boolean): Promise<IFluidRouter>;
+    getRootDataStore(id: string, wait?: boolean): Promise<IFluidDataStoreRuntimeEntrypoint>;
```

Comment:
we could leave this unchanged for now, if the caller (us) knows the returned object might be a `Partial<IProvideFluidHandle>` they can inspect and use it